win32/configure.bat

```diff
@@ -38,8 +38,12 @@ if "%1" == "--enable-devel" goto :enable-devel
 if "%1" == "--disable-devel" goto :disable-devel
 if "%1" == "--enable-rubygems" goto :enable-rubygems
 if "%1" == "--disable-rubygems" goto :disable-rubygems
+if "%1" == "--enable-jit-support" goto :enable-jit-support
+if "%1" == "--disable-jit-support" goto :disable-jit-support
 if "%1" == "--enable-mjit-support" goto :enable-mjit-support
 if "%1" == "--disable-mjit-support" goto :disable-mjit-support
+if "%1" == "--enable-mjit" goto :enable-mjit-support
+if "%1" == "--disable-mjit" goto :disable-mjit-support

```

Comment:
Having both --enable-mjit and --enable-jit-support feels confusing. Can we leave only --enable-mjit?

Reply:
That means deprecating `--enable-jit-support`?

Reply:
:+1: Yes, thanks. Now that we have `--enable-yjit`, it feels more consistent.